src/openzaak/conf/includes/api.py

```diff
@@ -1,12 +1,20 @@
 # SPDX-License-Identifier: EUPL-1.2
 # Copyright (C) 2019 - 2022 Dimpact
-from vng_api_common.conf.api import *  # noqa - imports white-listed
+from vng_api_common.conf.api import (  # noqa: F401
+    BASE_REST_FRAMEWORK,
+    BASE_SWAGGER_SETTINGS,
```

Comment:
I think we don't use it in OZ anymore, it's replaced with drf_spectacular settings